source/elements/oneTBB/source/algorithms/functions/parallel_for_each_func.rst

```diff
@@ -37,6 +37,7 @@ Function template that processes work items in parallel.
 Requirements:
 
 * The ``Body`` type must meet the :doc:`ParallelForEachBody requirements <../../named_requirements/algorithms/par_for_each_body>`.
+  Alternatively, since C++17 a pointer to member function in ``Index`` might be used as a ``Body``.
```

Comment:
Here and everywhere else, I think it should be "a pointer to **a** member function".

Reply:
Fixed